src/main/java/com/commercetools/project/sync/exception/NoStackTraceIllegalArgumentException.java

```diff
@@ -0,0 +1,18 @@
+package com.commercetools.project.sync.exception;
+
+/**
+ * Custom Exception class to return IllegalArgumentException without Stacktrace.
+ */
+public class NoStackTraceIllegalArgumentException extends IllegalArgumentException {
```

Comment:
The result looks like below when I run the main method directly without params,
````
[main] ERROR com.commercetools.project.sync.CliRunner - Failed to run sync process.
com.commercetools.project.sync.exception.NoStackTraceIllegalArgumentException: Please pass at least 1 option to the CLI.
```` 
maybe it's better to rename `NoStackTraceIllegalArgumentException` to  `CLIException` ? In that way, the user does not need to know your implementation details?  WDYT? 
````
[main] ERROR com.commercetools.project.sync.CliRunner - Failed to run sync process.
com.commercetools.project.sync.exception.CLIException: Please pass at least 1 option to the CLI.
````

Reply:
Done https://github.com/commercetools/commercetools-project-sync/pull/226/commits/18c74397cd1cfc0ff6c5107db1f581fa960567b2

Reply:
Just a minor opinion, personally i have no preference for the upper/lower case for abbreviation (e.g. HTTP or Http). But i see there is a class CliRunner in this repo. Shall we align the case ? @ahmetoz any opinion?

Reply:
Does not matter for me. CLI is better grammatically but for consistency, it might be CliException 😄

Reply:
@ahmetoz @leungkinghin , Done https://github.com/commercetools/commercetools-project-sync/pull/226/commits/de6bbb77c2ccb070861bca67ed53160dfe51a154. And I have removed "default" section in Switch statement, which was not reachable. Since it is already handled in the previous check. Please review these changes also.